### DISCUSSION TITLE
I didn't received my education benefits

### DISCUSSION THREAD

**Discussion by ratulhasanruhan** [community/community#132441] [GitHub Education]
### Hi everyone,

I already reverified my application with my varsity mail and ID card, but I haven't received a response yet.

**Comment by RajeevRanjan5642** [comment]
Hey 👋
If your application got approved, you will receive a mail of confirmation from github within 4-5 days and than you can avail the benefits.
Hope it helps 💖!

**Reply by ratulhasanruhan** [reply]
It's been 4-5 days already.
